Auth0/IDTokenValidator.swift

```diff
@@ -59,23 +59,20 @@ struct IDTokenValidator: JWTAsyncValidator {
 }
 
 enum IDTokenDecodingError: LocalizedError {
-    case missingToken
     case cannotDecode
 
     var errorDescription: String? {
         switch self {
-        case .missingToken: return "ID token is required but missing"
```

Comment:
There's no point in validating the ID Token presence if there will always be an ID Token in the response.